pages/index.js

```diff
@@ -1,18 +1,10 @@
 import BooksCard from '../component/booksCard';
-import styles from '../styles/Home.module.css';
 
-export function getServerSideProps() {
-  const API_KEY = process.env.REACT_APP_API_KEY;
-  return {
-    props: { API_KEY },
-  };
-}
-
-export default function Home({ API_KEY }) {
+export default function Home() {
   return (
     <>
       <h1>My Book App </h1>
-      <BooksCard apikey={API_KEY} />
+      <BooksCard />
```

Comment:
My personal preference: I think there is a better naming for this component in generel. Because it's not one card. So like `BooksCards` or `BooksList` maybe.